Library/Homebrew/keg_relocate.rb

```diff
@@ -174,16 +162,6 @@ def find_dylib(bad_name)
     lib.find { |pn| break pn if pn.to_s.end_with?(suffix) }
   end
 
-  def mach_o_files
-    mach_o_files = []
-    path.find do |pn|
-      next if pn.symlink? || pn.directory?
-      mach_o_files << pn if pn.dylib? || pn.mach_o_bundle? || pn.mach_o_executable?
-    end
-
-    mach_o_files
-  end
```

Comment:
I had to move this because of some strangeness with how the original `mach_data` worked. Instead of skirting around non-mach-o files, it raises an exception and then later rescues that exception to return an empty array. This messes with the `ruby-macho` calls, as they expect the array to be nonempty.

This could be fixed by guarding the usage of methods like `Pathname#dylib?` with a file sanity check, but until then I just decided to preserve the behavior that the original `mach_data` produced.